_tests/integration/secret_keys_sharing_test_bitrise.yml

```diff
@@ -0,0 +1,24 @@
+format_version: "11"
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+
+workflows:
+  secret-sharing:
+    steps:
+    - script:
+        title: This does not receive the secret keys
+        inputs:
+        - content: |-
+            #!/bin/env bash
+            set -x # Do not set -e as bitrise command is expected to fail
```

Comment:
Maybe I just don't understand the feature, but what bitrise command fails in this script?

Reply:
Ahh yes, that is just a leftover from my previous testing. I will remove it.